jujugui/static/gui/src/app/components/env-switcher/env-switcher.js

```diff
@@ -249,16 +249,21 @@ YUI.add('env-switcher', function() {
 
     render: function() {
       return (
-        <div className="env-switcher">
-          <div
+        <div className="env-switcher" aria-label="Environment switcher">
+          <a
```

Comment:
It looks like this causes the GUI to change routes when the env switcher is opened.